docker/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM node:16
+# LABEL org.opencontainers.image.authors="Mogztter#Guillaume Grossetie"
+LABEL version="15.4.0"
```

Comment:
I think we should remove it, the version will be the tag name (and it will most likely be the asciidoctor-web-pdf version)

Reply:
Ok.